docs/how-to-guides/integrating-autoware/creating-vehicle-and-sensor-description/creating-sensor-description/creating-sensor-description.md

```diff
@@ -0,0 +1,867 @@
+of the vehicle is parent.
+
+### 2. sensors_calibration.yaml
+
+A file that defines the mounting positions and orientations of sensors with `base_link` as the parent frame.
+
+The original file is [sensors_calibration.yaml](https://github.com/autowarefoundation/sample_sensor_kit_launch/blob/main/sample_sensor_kit_description/config/sensors_calibration.yaml) included in [sample_sensor_kit_launch](https://github.com/autowarefoundation/sample_sensor_kit_launch/tree/main).
+
+??? note "sensors_calibration.yaml for sample_sensor_kit_launch"
+
+    ```yaml
+    base_link:
+      sensor_kit_base_link:
+        x: 0.9
+        y: 0.0
+        z: 2.0
+        roll: -0.001
+        pitch: 0.015
+        yaw: -0.0364
+        .
+        .
+        .
+    ```
+
+The frame given as `base_link` in the above file can be defined as point 0 of the vehicle.
+For this point, the desired sensor frame or any point on the vehicle can be selected,
+and the most suitable point is the point defined as the "axle,"
+which is the middle of the two rear wheels of the vehicle.
+
+The tutorial_vehicle file is [sensors_calibration.yaml](https://github.com/leo-drive/tutorial_vehicle_sensor_kit/blob/main/tutorial_vehicle_sensor_kit_description/config/sensors_calibration.yaml) included in [tutorial_vehicle_sensor_kit](https://github.com/leo-drive/tutorial_vehicle_sensor_kit/tree/main) and given below.
+In sensors_calibration.yaml file, parent_frame's position relative to base_link is given in euler format as [x, y, z, roll, pitch, yaw].
+
+??? note "sensors_calibration.yaml for tutorial_vehicle_sensor_kit_launch"
+
+    ```yaml
+    base_link:
+      sensor_kit_base_link:
+        x: 0.0
+        y: 0.0
+        z: 0.0
+        roll: 0.0
+        pitch: 0.0
+        yaw: 0.0
+    ```
+
+In this example file, the top_lidar in the center of the vehicle is selected as the parent_frame, and it is prepared in such a way that the position of the parent_frame element relative to the axle of the vehicle is entered.
+The [x, y, z, roll, pitch, yaw] values in this file can be entered by base_link - lidar calibration or by measuring from the 3D model.
+
+### 3. sensor_kit.xacro
+
+Resolves the positions of sensors with sensor_kit_base_link as the parent and defines the positions and orientations based on sensor_kit_calibration.yaml in `individual_params`.
+
+The original file is [sensor_kit.xacro](https://github.com/autowarefoundation/sample_sensor_kit_launch/blob/main/sample_sensor_kit_description/urdf/sensor_kit.xacro)
+included in [sample_sensor_kit_launch](https://github.com/autowarefoundation/sample_sensor_kit_launch/tree/main).
```

Comment:
Since sensor_kit.xacro is big file, we can hide by using `note` element.
```suggestion
The defaul [sensor_kit.xacro](https://github.com/autowarefoundation/sample_sensor_kit_launch/blob/main/sample_sensor_kit_description/urdf/sensor_kit.xacro)
 file for sample_sensor_kit is given below:
```